lib/resolver/homebrew-file.js

```diff
@@ -3,6 +3,6 @@ import relativeFile from './relative-file.js';
 export default function ({ path, target }) {
   return [
     relativeFile({ path, target }),
-    'https://github.com/Homebrew/homebrew-core/blob/master/Formula/' + target,
+    '{BASE_URL}/Homebrew/homebrew-core/blob/master/Formula/' + target,
```

Comment:
Pushed a new commit where this gets fixed 4d2e7b2e05c5dd98aabb6e1572cf9c316418952b